features/size_validation.feature

```diff
@@ -89,46 +99,51 @@ Feature: Applicant uploads too large a file for their application
 
     And I should see t("shf_applications.update.success")
 
-
+  @selenium
   Scenario: Existing application - Uploads a file just under the size limit
     Given I am logged in as "emma@happymutts.se"
     And I am on the "edit my application" page
     And I choose a file named "upload-just-under-limit.pdf" to upload
+    And I select files delivery radio button "upload_now"
     When I click on t("shf_applications.edit.submit_button_label")
     Then I should not see t("activerecord.errors.models.uploaded_file.attributes.actual_file_file_size.file_too_large")
     And I should see t("shf_applications.update.success")
 
-
+  @selenium
   Scenario: Size error message in English when locale = en
     Given I am logged in as "emma@happymutts.se"
     And I set the locale to "en"
     And I am on the "edit my application" page
     And I choose a file named "diploma_huge.pdf" to upload
+    And I select files delivery radio button "upload_now"
     When I click on t("shf_applications.edit.submit_button_label")
     Then I should not see t("activerecord.errors.models.uploaded_file.attributes.actual_file_file_size.file_too_large", locale: :sv)
     And I should see t("activerecord.errors.models.uploaded_file.attributes.actual_file_file_size.file_too_large", locale: :en)
     And I set the locale to "sv"
 
-
+  @selenium
   Scenario: Size error message in sveriges when locale = sv
     Given I am logged in as "emma@happymutts.se"
     And I set the locale to "sv"
     And I am on the "edit my application" page
     And I choose a file named "diploma_huge.pdf" to upload
+    And I select files delivery radio button "upload_now"
```

Comment:
see above; this should come before previous step

Reply:
It doesn't need to, since the user can select files for uploading, and _then_ select the file delivery method (which will probably happen more often than not since they may not pay attention to the selection until they notice the "save" button is disabled).